src/logic/embedding.lean

```diff
@@ -83,6 +83,16 @@ protected def congr {α : Sort u} {β : Sort v} {γ : Sort w} {δ : Sort x}
   (e₁ : α ≃ β) (e₂ : γ ≃ δ) (f : α ↪ γ) : (β ↪ δ) :=
 (equiv.to_embedding e₁.symm).trans (f.trans e₂.to_embedding)
 
+/-- Embeddings are equivalent under equivalences. -/
+protected def equiv {α : Sort u} {β : Sort v} {γ : Sort w} {δ : Sort x}
```

Comment:
I think this should probably be called `equiv.embedding_congr`, since it's essentially `equiv.arrow_congr` but for `↪` instead of `→`. You should probably add the four `_comp`, `_refl`, `_trans`, and `_symm` lemmas that `arrow_congr` has to go with it too.

Reply:
alright, will get on with that!

Reply:
okay, so that's been done in the latest commit, but I have a couple followup q's/notes:

* is it worth putting `@[congr, simps apply]` on this def too? I'm shaky on how both of them work; I put them in as a parallel to `arrow_congr`.
* `arrow_congr` uses `Sort*`, whilst `logic.embedding` tends to have universe args. I went with `Sort*`; does it matter?
* `equivalence` has no non-coe `comp` (unless I'm blind, which wouldn't surprise me) so I skipped `embedding_congr_comp`,

Reply:
> equivalence has no non-coe comp

I assume you meant `embedding`? It does, but confusingly it's called `trans`. So perhaps that lemma would be called `embedding_congr_apply_trans` to distinguish it from `embedding_congr_trans`?